packages/datadog_flutter_plugin/example/android/build.gradle

```
@@ -16,7 +16,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.0.3'
+        classpath 'com.android.tools.build:gradle:7.4.0'
```

minor: the latest is `7.4.1`

```suggestion
        classpath 'com.android.tools.build:gradle:7.4.1'
```